web/views.py

```
@@ -1242,6 +1244,49 @@
     return HttpResponse(status=200)
 
 
+def crowdfunding_detail(request, campaign_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `crowdfunding_detail`

(ANN201)